molecule/resources/prepare.yml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
Why do we need that?

Reply:
Now looking into this i can agree that we can use out of the box driver, I'll additionally test it.

Reply:
I was wrong, we need modified prepare.yml, it waits for ec2 instance to reboot after additional steps provided as user_data to ec2 module.